### PR TITLE
internal/admin: codify project configuration into Terraform

### PR DESCRIPTION
**Pull request by zombiezen**
This allows us to version control and review changes to our repository configuration just like other changes, as well as keep both Wire and Go Cloud mostly in sync in terms of setup. Testing this already corrected multiple such discrepancies between the projects, so this seems like a good thing to have.

### CODE REVIEW

File: internal/admin/main.tf [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+provider "google" {
+  version = "~> 1.15"
+}
+
+module "go_cloud_repo" {
+  source       = "./repository"
+  name         = "go-cloud"
+  description  = "A library and tools for open cloud development in Go."
+  homepage_url = "https://godoc.org/github.com/google/go-cloud"
```
Comment (jba) [inline internal/admin/main.tf]: godoc.org/gocloud.dev

Comment (zombiezen) [reply internal/admin/main.tf]: Done. And this is why I like having code-reviewed configurations. 🙂

**Comment by zombiezen** [comment]
@jba, I need your approval before merging.

**Comment by jba** [comment]
Sorry!

**Pull request merged**